Source/Core/Core/Boot/Boot_BS2Emu.cpp

```diff
@@ -227,7 +227,7 @@ bool CBoot::SetupWiiMemory()
   SettingsHandler gen;
   std::string serno;
   const std::string settings_file_path(
-      Common::GetTitleDataPath(Titles::SYSTEM_MENU, Common::FROM_SESSION_ROOT) + WII_SETTING);
+      Common::GetTitleDataPath(Titles::SYSTEM_MENU, Common::FROM_SESSION_ROOT) + "/" + WII_SETTING);
```

Comment:
`std::string`s are still going to be constructed in these cases, so I don't think there's an advantage to using `'/'` here. I can still change it if you want though

Reply:
You can keep it like it is if you want to.

Reply:
Generally it's cheaper to append a character literal over a string literal (don't need to run the character literal to check for the total characters being appended). But given this in a setup function, I don't think it matters too much.

(technically if we wanted to go full-on performance, we'd use `.append()` or `operator+=` instead of `operator+` for strings, because `append()` just extends the first string's buffer instead of constructing and discarding temporary `std::string` instances like what `operator+` does when chained together)

Reply:
Oh, that's good to know. I'll keep this in mind for future changes.